oc-admin/about.php

```diff
@@ -11,143 +11,132 @@
-									<p>
-										<em>Note:</em> Used to encrypt cookie 'aljksdz7' and authenticate
-										request to the api if the requestor is not logged in.
-									</p>
-									<a style="margin-left:10px" class="btn btn-primary" href="<?php echo BASE_URL; ?>/actions/generalActions.php?newApiKey=1">Generate</a>
-								</div>
+							<div class="form-group">
+								<label for="name"><?php echo lang_key("DATABASE_ENGINE"); ?></label>
+								<input type="text" class="form-control" readonly="readonly" value="<?php echo getMySQLVersion(); ?>" />
+								<?php echo lang_key("DATABASE_ENGINE_notes"); ?>
 							</div>
-						</div>
-						<div class="card">
-							<div class="card-header">
-								<em class="fa fa-align-justify"></em> <?php echo lang_key("ABOUT_OPENCAD"); ?>
+							<div class="form-group">
+								<label for="name"><?php echo lang_key("LOADED_PHP_MODULES"); ?></label>
+								<input type="text" class="form-control" readonly="readonly" placeholder="<?php
+																											$modules = get_loaded_extensions();
+																											foreach ($modules as $module) {
+																												echo $module . ", ";
+																											} ?>" />
+								<?php echo lang_key("LOADED_PHP_MODULES_notes"); ?>
 							</div>
-							<div class="card-body">
-								<p>OpenCAD is an open source project licensed under GNU GPL v3. The original code and concept by <a rel="noopener" href="https://github.com/ossified" title="a link to the original developer's GitHub.">Shane Gill</a>. This project is maintained by Overt Source.</p>
+							<div class="form-group">
+								<label for="name"><?php echo lang_key("API_KEY"); ?></label>
+								<input type="text" class="form-control" readonly="readonly" placeholder="<?php echo getApiKey(); ?>" />
+								<p>
+									<em>Note:</em> Used to encrypt cookie 'aljksdz7' and authenticate
+									request to the api if the requestor is not logged in.
+								</p>
+								<a style="margin-left:10px" class="btn btn-primary" href="<?php echo BASE_URL; ?>/actions/generalActions.php?newApiKey=1">Generate</a>
 							</div>
 						</div>
 					</div>
-				</main>
-			</div>
-	<?php require_once ( ABSPATH . "/" . OCTHEMES ."/". THEME ."/includes/footer.inc.php"); ?>
-	<?php require_once( ABSPATH . OCTHEMEINC ."/admin/topbarNav.inc.php" ); ?>
-	<?php require_once( ABSPATH . OCTHEMEINC ."/scripts.inc.php" ); ?>
-	</body>	
+					<div class="card">
+						<div class="card-header">
+							<em class="fa fa-align-justify"></em> <?php echo lang_key("ABOUT_OPENCAD"); ?>
+						</div>
+						<div class="card-body">
+							<p>OpenCAD is an open source project licensed under GNU GPL v3. The original code and concept by <a rel="noopener" href="https://github.com/ossified" title="a link to the original developer's GitHub.">Shane Gill</a>. This project is maintained by Overt Source.</p>
```

Comment:
We can probably drop the the reference to Overt Source. I'll get clarification on this with @phillf.

```suggestion
							<p>OpenCAD is an open source project licensed under GNU GPL v3. The original code and concept by <a rel="noopener" href="https://github.com/ossified" title="a link to the original developer's GitHub.">Shane Gill</a>.</p>
```